config/routes.rb

```diff
@@ -4,7 +4,14 @@
   resources :backgrounds
   resources :reports
   resources :pipeline_outputs, only: [:index, :show]
-  devise_for :users
+  devise_for :users, controllers: {
+    sessions: 'sessions',
+    registrations: 'registrations'
+  }
+  devise_scope :user do
```

Comment:
why do we need this? 
why not just use /users/sign_in, /users/sign_out?

Reply:
I figured it is easier go to login, instead of the longer route? @yunfangjuan. It renders the same view.

Reply:
but it's redudant and makes code management more complicated.